policy_set_integration_test.go

```diff
@@ -154,6 +154,34 @@ func TestPolicySetsCreate(t *testing.T) {
 		assert.Equal(t, ps.Workspaces[0].ID, wTest.ID)
 	})
 
+	t.Run("with policies and projects provided", func(t *testing.T) {
+		skipUnlessBeta(t)
+		pTest, pTestCleanup := createPolicy(t, client, orgTest)
+		defer pTestCleanup()
+		wTest, wTestCleanup := createWorkspace(t, client, orgTest)
+		defer wTestCleanup()
+		prjTest, prjTestCleanup := createProject(t, client, orgTest)
+		defer prjTestCleanup()
+
+		options := PolicySetCreateOptions{
+			Name:       String("populated-policy-set"),
+			Policies:   []*Policy{pTest},
+			Workspaces: []*Workspace{wTest},
+			Projects:   []*Project{prjTest},
+		}
+
+		ps, err := client.PolicySets.Create(ctx, orgTest.Name, options)
+		require.NoError(t, err)
+
+		assert.Equal(t, ps.Name, *options.Name)
+		assert.Equal(t, ps.PolicyCount, 1)
+		assert.Equal(t, ps.Policies[0].ID, pTest.ID)
+		assert.Equal(t, ps.WorkspaceCount, 1)
+		assert.Equal(t, ps.Workspaces[0].ID, wTest.ID)
+		assert.Equal(t, ps.ProjectCount, 1)
+		assert.Equal(t, ps.Projects[0].ID, prjTest.ID)
```

Comment:
Doesn't affect the test but, for .Equal the 2nd argument is expected and 3rd argument is the actual value.
eg. assert.Equal(t, 1, ps.ProjectCount)
But, now that I look at this, this is not followed in the entire file lol, so I don't know we want to change it now.

Reply:
I've kept it for consistency. It is important for both the values to match which would be asserted this way as well.